rdmo/management/management/commands/merge_attributes.py

```diff
@@ -0,0 +1,249 @@
+import logging
+from textwrap import dedent
+
+from django.core.management.base import BaseCommand, CommandError
+
+from rdmo.domain.models import Attribute
+from rdmo.management.utils import replace_uri_in_template_string
+from rdmo.projects.models import Value
+from rdmo.views.models import View
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = 'Replace an attribute with another attribute across multiple models'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--source',
+            required=True,
+            help='The URI of the source attribute that will be replaced by the target and will be deleted'
+        )
+        parser.add_argument(
+            '--target',
+            required=True,
+            help='The URI of the target attribute that will be used to replace the source'
+        )
+        parser.add_argument(
+            '--save',
+            action='store_true',
+            default=False,
+            help='''If specified, the changes will be saved.\
+                    If not specified, the command will not make any changes in the database.'''
+            )
+        parser.add_argument(
+            '--delete',
+            action='store_true',
+            default=False,
+            help='''If specified, the source attribute will be deleted.\
+                    If not specified, the command will not delete the source attribute.'''
+            )
+        parser.add_argument(
+            '--view',
+            action='store_true',
+            default=False,
+            help='If specified, the changes will be applied to the affected Views as well.'
+        )
+
+    def handle(self, *args, **options):
+        source = options['source']
+        target = options['target']
+        save_changes = options['save']
+        delete_source = options['delete']
+        update_views = options['view']
+        verbosity = options.get('verbosity', 1)
+
+        source = get_valid_attribute(source, message_name='Source')
+        target = get_valid_attribute(target, message_name='Target')
```

Comment:
we ran into minor issue with this command. Currently, the target attribute can not be an attribute that has descendants but this condition matters only for the source and not for the target. 
I would like to change the `get_valid_attribute` function and only check the `attribute.get_descendants()` when eg. `allow_descendants` is not true

Reply:
go for it.